doc/experiment.rst

```diff
@@ -259,4 +259,58 @@ the following:
 
   cd /scratch/smartsim/Training-Run ; /usr/bin/srun --output /scratch/smartsim/Training-Run/Training-Ensemble_3.out --error /scratch/smartsim/Training-Ensemble_3.err --job-name Training-Ensemble_3-CHTN0UI2TRE7 --nodes=1 --ntasks=24 /scratch/pyenvs/smartsim/bin/python ./train-model.py --LR=0.35000000000000003 &
 
-  wait
\ No newline at end of file
+  wait
+
+Prefixing Keys in the Orchestrator
+----------------------------------
+
+If each of multiple ensemble members attempt to use the same code access their respective models in
+the Orchestrator, the keys by which they do this will overlap and they can end up accessing each
+others' models inadvertently. To prevent this situation, the SmartSim Entity object supports key
+prefixing, which automatically prepends the name of the model to the keys by which it is accessed.
+With this enabled, key overlapping is no longer an issue and ensemble members can use the same code.
```

Comment:
This is a good update and example.  One area where we might want to dive deeper is that users are getting tripped up on ``enable_key_prefixing`` only setting outbound key prefixing (i.e. ``SSKEYOUT``) and they would need to register incoming connections (including self) to be able to pull keys from the ensemble member namespace.  

Also, we should double check on the default client prefixing behavior with ``SSKEYOUT`` and ``SSKEYIN``.  I think there are some subtitles to prefixing behavior between tensors, models, datasets, aggregation lists, and scripts.  Maybe deeper content around this behavior belongs here and in the clients (or a link between the two)